README.md

```diff
@@ -12,96 +11,110 @@
+* Explain what the above code does in their own words:
 
 ```swift
 // We declared a variable named fourthPlanet that is of type String
 ```
 * Explain the difference between type inference and type annotations. 
-* Create a variable of type Int
-
-
+* Create a variable of type `Int`
 
 ## What the student can do at this point 
 
 * Create variables and constants
-* Use the print function to print strings to console
-* Can distinguish where it's appropriate to use let over var
-* A student has NOT seen the following syntax (yet) using type annotations.
+* Use the `print` function to print strings to console
+* Can distinguish where it's appropriate to use `let` over `var`
 
-```swift
-let name: String
-name = "Jim"
+## Strings
 
-let movieTitle: String = "Wall-E"
+So far in your lessons, you've only been working with strings. "String" is a generic programming term for a sequence of characters. In Swift, the _data type_ that represents a string is also called a `String` (with a capital S). Swift strings are also sequences of characters. Often times, these characters are letters and numbers, and `String`s are recognizable as words and sentences. However, in Swift, a `String` can contain any character. In addition to alphanumeric characters like `a` and `9`, they can also be symbols such as `@` or `+`, or even a space. They can also be characters from languages other than English, such as `ñ` or `ß`. They can even contain other symbols, like `™` or `©`. If you can type it, you can put it in a `String`!
+
+Let's investigate types. Create a new playground file and add this as the first line:
+
+```swift
+let thirdPlanet = "Earth"
 ```
 
+What type is the constant `thirdPlanet`? If you guessed `String`, you're right!
+
+Variables and constants in Swift are always associated with a type. In programming languages, a _type_ describes what kind of thing a variable or constant is. More importantly, a variable's type dictates how it behaves and what actions you can perform on it.
 
+Consider two different kinds of things in the real world: a piece of paper and an aluminum can. You do things to a piece of paper that you would never do to a can, such as writing on it or crumpling it into a ball. You can't crumple a can (well, unless you smash it against your forehead, but that will hurt and you shouldn't do that)! Likewise, you do things to a can that you wouldn't do to a piece of paper, like drink out of it.
 
-## Outline / Notes
+In programming languages, including Swift, variables and constants also have types that dictate the things you can do to them. You may remove blank spaces from the beginning and end of a string, which makes no sense on a number (they don't have blank spaces). Or you may divide a number by 10, but it doesn't make any sense to divide a string by 10. Swift keeps track of the types of variables and prevents you from doing things to them that don't make sense, which will help prevent errors from occurring in your programs.
 
```

Comment:
I *LOVE* this analogy. I'm a huge fan of doing exactly this with the students here.